File: server/dna.core/dna.core.service/Models/ImageModel.cs
using dna.core.service.Models.Abstract;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace dna.core.service.Models
{
    public class ImageModel : WriteHistoryBaseModel, IModelBase
    {
        public int Id { get; set; }

        [Required(ErrorMessage = "Required")]
        public string ImageUrl { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(100, ErrorMessage = "StringLength")]
        public string FileName { get; set; }

        [Required(ErrorMessage = "Required")]
        [StringLength(4, ErrorMessage = "StringLength")]
        public string FileExtension { get; set; }

        [Required(ErrorMessage = "Required")]
        public bool IsPrimary { get; set; }

        public string Description { get; set; }
    }
}

File: server/medicore.apps/src/model/imageModel.ts
import { IEntityBase } from './IEntityBase';
import { WriteHistoryBaseModel } from './writeHistoryBaseModel';

export class ImageModel extends WriteHistoryBaseModel implements IEntityBase {
    id: number;
    imageUrl: string;
    fileName: string;
    fileExtension: string;
    isPrimary: boolean;
    description: string;
}